content/ko/community/_index.html

```diff
@@ -0,0 +1,236 @@
+<div class="videocta">
+커뮤니티 맴버와의 대화을 봅니다.&nbsp;&#9654;
+</div>
+
+</a>
+<a name="discuss"></a>
+</div>
+</div>
+</div>
+
+
+<div class="resources">
+<br class="mobile"><br class="mobile">
+<br class="tablet"><br class="tablet">
+<h1 style="padding-top:1%">토론</h1>
+
+<div style="font-weight:300;text-align:center">우리는 대화를 많이 합니다. 우리를 찾아 이 모든 플랫폼에서의 대화에 참여하세요.</div>
+
+<div class="resourcecontainer">
+
+<div class="resourcebox">
+<img src="/images/community/discuss.png" alt=Forum" style="width:80%;padding-bottom:2%">
+<a href="https://discuss.kubernetes.io/" style="color:#0662EE;display:block;margin-top:1%">
+포럼&nbsp;&#9654;
+</a>
+<div class="resourceboxtext" style="font-size:12px;text-transform:none !important;font-weight:300;line-height:1.4em;color:#333333;margin-top:4%">
+문서, 스택오버플로우 등을 연결하는 주제 기반 기술 토론하는 곳입니다.
+</div>
+</div>
+
+<div class="resourcebox">
+<img src="/images/community/twitter.png" alt="Twitter" style="width:80%;padding-bottom:2%">
+<a href="https://twitter.com/kubernetesio" style="color:#0662EE;display:block;margin-top:1%">
+트위터&nbsp;&#9654;
+</a>
+<div class="resourceboxtext" style="font-size:12px;text-transform:none !important;font-weight:300;line-height:1.4em;color:#333333;margin-top:4%">블로그 게시물, 이벤트, 뉴스, 아이디어의 실시간 소식들입니다.
+</div>
+</div>
+
+<div class="resourcebox">
+<img src="/images/community/github.png" alt="GitHub" style="width:80%;padding-bottom:2%">
+<a href="https://github.com/kubernetes/kubernetes" style="color:#0662EE;display:block;margin-top:1%">
+깃헙&nbsp;&#9654;
+</a>
+<div class="resourceboxtext" style="font-size:12px;text-transform:none !important;font-weight:300;line-height:1.4em;color:#333333;margin-top:4%">
+모든 프로젝트와 이슈 추적, 더불어 코드도 물론이죠.
+</div>
+</div>
+
+<div class="resourcebox">
+<img src="/images/community/stack.png" alt="Stack Overflow" style="width:80%;padding-bottom:2%">
+<a href="https://stackoverflow.com/search?q=kubernetes" style="color:#0662EE;display:block;margin-top:1%">
+스택 오버플로우&nbsp;&#9654;
+</a>
+<div class="resourceboxtext" style="font-size:12px;text-transform:none !important;font-weight:300;line-height:1.4em;color:#333333;margin-top:4%">
+  모든 유스 케이스에 대한 기술적인 문제 해결입니다.
+  <a name="events"></a>
+</div>
+</div>
+
+<!--
+<div class="resourcebox">
+
+<img src="/images/community/slack.png" style="width:80%">
+
+slack&nbsp;&#9654;
+
+<div class="resourceboxtext" style="font-size:11px;text-transform:none !important;font-weight:200;line-height:1.4em;color:#333333;margin-top:4%">
+170개 이상의 채널이 있으며, 필요에 맞는 채널을 찾을 수 있습니다.
+</div>
+
+</div>-->
+
+</div>
+</div>
+<div class="events">
+  <br class="mobile"><br class="mobile">
+  <br class="tablet"><br class="tablet">
+<div class="eventcontainer">
+ <h1 style="color:white !important">다가오는 이벤트</h1>
+    {{< upcoming-events >}}
+</div>
+</div>
+
+<div class="meetups">
+<div class="meetupcol">
+<div class="meetuptext">
+<h1 style="text-align:left">글로벌 커뮤니티</h1>
+전 세계에는 150개가 넘는 모임이 있고 성장하고 있는 가운데, 현지 kube 사람들을 찾아보세요. 가까운 곳에 없다면 책임을 지고 여러분만의 것을 만들어보세요.
```

Comment:
```suggestion
전 세계 150개가 넘는 모임이 있고 성장하고 있는 가운데, 현지 kube 사람들을 찾아보세요. 가까운 곳에 없다면, 책임감을 갖고 직접 만들어보세요.
```

위와 같이 제안드립니다. 참고 바랍니다.